ffi/examples/read-table/read_table.c

```diff
@@ -167,6 +168,199 @@ void free_partition_list(PartitionList* list) {
   free(list);
 }
 
+
+// Our schema code might add (is nullable: x) to names, which we need to strip off.
+// Returns the location where the edit was made if one was made, or NULL if no edit
+char* fixup_name(char* name) {
```

Comment:
this is kinda a weird edge case but does this break if the name is something like `(name)`?